starport/services/network/networkchain/init.go

```diff
@@ -11,40 +11,40 @@ import (
 
 // Init initializes blockchain by building the binaries and running the init command and
 // create the initial genesis of the chain, and set up a validator key
-func (c *Chain) Init(ctx context.Context) error {
+func (c *Chain) Init(ctx context.Context) (err error) {
```

Comment:
If feel like we can avoid named errors like this in big funcs similar to this one. it's a bit hard to maintain